docs/ansible/roles/slapd/examples/multi-master-replication.yml

```diff
@@ -69,7 +69,7 @@ slapd__cluster_tasks:
              tls_key="{{ slapd__tls_private_key }}"
              tls_cipher_suite="{{ slapd__tls_cipher_suite }}"
 
-      olcMirrorMode: 'TRUE'
+      olcMultiProvider: 'TRUE'
```

Comment:
Does this also work on older `slapd` releases, ie 2.4 from older Debian versions? Ah, I guess not.  Hmmm... To make things easy for migration, we could duplicate the sections and use conditional state with `exact` and `ignore` for the specific `slapd` version. The version should be available in the `slapd__version` variable. After 1-2 Debian releases, duplicate entries for old versions could be removed.

Reply:
@drybjed I would tend to disagree....this is a documentation example...and the patch also adds a notice that the name has changed for older releases....adding complex conditionals just means we'll have to revisit this later to cleanup the conditionals (and we all know how likely that is....)